x-pack/plugins/ml/public/components/anomalies_table/_anomalies_table.scss

```diff
@@ -111,12 +111,17 @@
         margin-top: 0px;
         flex-basis: 15%;
         font-size: inherit;
+        line-height: 1.5rem;
+        text-overflow: ellipsis;
+        overflow: hidden;
+        white-space: nowrap;
```

Comment:
We also have a mixin helper for truncating text that could help you out here. Instead of needing to write these 3 lines, you can just include `@include euiTextTruncate`.